joomla/joomla-cms
# [4.0] accessible validation

Pull Request for Issue # .

### Summary of Changes
Joomla 4 introduced inline validation messages per field. Unfortunately the implementation was no accessible. This PR fixes that. Also Improves accessibility to the administration login page

- Inline validation messages need to be part of the label

### Testing Instructions
Apply PR, Logout try to login in administration area, without filling the username and password, or try to tab through the fields.

Create a new article and try to save it without filling a title.

The error message should be **focused**. If you have a mac turn on VoiceOver and redo the previous step: you should hear the error message!


### Preview
![screen shot 2017-07-28 at 17 48 54](https://user-images.githubusercontent.com/3889375/28723190-ab010ce0-73be-11e7-950d-41ff86e702cb.png)
![screen shot 2017-07-28 at 17 48 26](https://user-images.githubusercontent.com/3889375/28723193-ade6e4a2-73be-11e7-86a8-b7f4f3eac117.png)




### Documentation Changes Required
None
